docker-compose.yml

```diff
@@ -11,9 +11,9 @@ services:
       - "80:80"
       - "443:443"
     volumes:
-      - /etc/basic_auth/.htpasswd:/etc/basic_auth/.htpasswd
-      - /tf_aarch64/volumes/builds:/www/media/builds
-      - /tf_aarch64/volumes/logs:/www/media/logs
+      - ./basic_auth/.htpasswd:/etc/basic_auth/.htpasswd
```

Comment:
These changes will break the current webserver setup. But I guess it is okay.

Reply:
I see, what should I do to set up the webserver correctly?

Reply:
Copy the contents of /tf_aarch64 to /tmp/tf_aarch64 and /etc/basic_auth/.httpasswd to ./basic_auth. I will do so myself. I am talking about [this server](http://34.136.139.255/) btw.